CppCoreGuidelines.md

```diff
@@ -16867,6 +16867,47 @@ Static helps dynamic: Use static polymorphism to implement dynamically polymorph
 Dynamic helps static: Offer a generic, comfortable, statically bound interface, but internally dispatch dynamically, so you offer a uniform object layout.
 Examples include type erasure as with `std::shared_ptr`'s deleter (but [don't overuse type erasure](#Rt-erasure)).
 
+    #include <memory>
```

Comment:
could you start the example with "##### Example"?
also I still feel the naming of Concept and concept_ is distractingly close to the keyword concept

Reply:
@cubbimew The example is already in an example (the line before "Dynamic helps static...").

I change the name of classes to be the same than the "Static helps dynamic" example.

Reply:
great, thanks!